src/main/java/net/unicon/lti/controller/lti/LTI3Controller.java

```diff
@@ -299,12 +299,12 @@ public String error(HttpServletRequest req) {
             log.debug("Timeout has occurred.");
             log.debug("middlewareState: " + req.getParameter("middlewareState"));
             log.debug("lmsTimedOut: " + req.getParameter("lmsTimedOut"));
-            return "lti3safarierror";
+            return "lti3ErrorMessage";
```

Comment:
I replaced the lti3safari error here with the new lti 3 error. Please let me know if this shouldn't be the case or if the new error message should be placed elsewhere.

Reply:
I think that's right--last we chatted about it I was under the impression there would be one error state for every error in the middleware.

Reply:
Actually, this call to lti3safarierror _should_ have been changed to lti3ErrorMessage - or, now, just error - because we do want to see the real error page here.  Apologies for the confusion!

Reply:
Yep that makes sense to me, we just want to retain the `lti3safarierror` on the `OIDCController` for moodle, but the `lti3safarierror` calls here in the `LTI3Controller` can switch to using `error` now. Sorry for the back and forth on that Annalyn!

Reply:
no problem! thank you all for checking and the helpful comments!